c2rust-build-paths/src/lib.rs

```diff
@@ -128,3 +128,17 @@ pub fn find_llvm_config() -> Option<PathBuf> {
             .next()
         })
 }
+
+pub fn invoke_command<I, S>(command: Option<&Path>, args: I) -> Option<String>
+where
+    I: IntoIterator<Item = S>,
+    S: AsRef<OsStr>,
+{
+    let command = command?;
+    let output = Command::new(command).args(args).output().ok()?;
+    if output.status.success() {
+        Some(String::from_utf8(output.stdout).unwrap().trim().to_owned())
+    } else {
+        None
+    }
+}
```

Comment:
I would just inline this.

Reply:
```rust
let llvm_config = find_llvm_config().expect("llvm-config not found");
let output = Command::new(llvm_config)
    .args(&["--bindir"])
    .output()
    .ok()
    .filter(|output| output.status.success())
    .expect("llvm-config error");
String::from_utf8(output.stdout).unwrap().trim().to_owned()
```

Reply:
fixed in [828243d](https://github.com/immunant/c2rust/pull/767/commits/828243d339c6993bc0274865ccb3a512af765212)